test/bin/scenario.sh

```diff
@@ -262,39 +278,13 @@ wait_for_greenboot() {
 
     local -r start_time=$(date +%s)
     local -r ssh_cmd="ssh -oConnectTimeout=10 -oBatchMode=yes -oStrictHostKeyChecking=accept-new redhat@${ip}"
-    local retry_count=2
     while [ $(( $(date +%s) - start_time )) -lt "${VM_BOOT_TIMEOUT}" ] ; do
         local svc_state
-
         svc_state="$(${ssh_cmd} systemctl show --property=SubState --value greenboot-healthcheck || true)"
         if [ "${svc_state}" = "exited" ] ; then
             record_junit "${vmname}" "greenboot-check" "OK"
             return 0
         fi
-
-        # Print the last log and check for terminal failure
-        ${ssh_cmd} "sudo journalctl -n 10 -u greenboot-healthcheck" || true
```

Comment:
I agree, but this was removed on purpose to try to detect if the issue is really the registry or there is something else. If we leave this code it might go silent and there is no alert about it.